components/operator/config/rbac/role.yaml

```diff
@@ -1,9 +1,10 @@
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
```

Comment:
Are you sure it works?
The operator use some cluster scoped resources.

Reply:
Didn't sure .. I take a look why Kubebuilder changed it